tasks/main.yml

```diff
@@ -11,6 +11,8 @@
     backup: false
     dest: /etc/logrotate.d/haproxy
     src: logrotated-haproxy.j2
+  tags:
+    - haproxy-configure
```

Comment:
I understand why this could be useful, but this is just one of our many openmicroscopy roles and playbooks so I'd prefer not to add tags to individual roles unless we have an overall plan.

Reply:
I understand, I'll keep it in my own code.

Reply:
Btw, do you know when/if you will have such a plan?
That tag name actually comes from @geerlingguy's original role.

Reply:
I'm afraid we don't have a timeline yet.